### ISSUE TITLE
NTR: History of delivery by cesarean section  (General history, localization & modifiers)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7859]
New term request History of delivery by cesarean section  (brumme_220713134241): C-section, laparotomy and hysterotomy  UMLS: 1384674

**Comment by pnrobinson** [comment]
This is the intended meaning of the term

[Caesarian section HP:0011410](https://hpo.jax.org/app/browse/term/HP:0011410)

Maybe it would be better to change the primary label?

**Comment by pnrobinson** [comment]
@MickeySegal
Not sure you saw the above comment?

**Comment by MickeySegal** [comment]
This is good.  I don't think I had seen this.

**Comment by pnrobinson** [comment]
thanks, closing
